internal/cri/server/restart.go

```diff
@@ -106,6 +107,14 @@ func (c *criService) recover(ctx context.Context) error {
 		metadata := sandboxstore.Metadata{}
 		err := sbx.GetExtension(podsandbox.MetadataKey, &metadata)
 		if err != nil {
+			if errors.Is(err, errdefs.ErrNotFound) {
+				err = c.client.SandboxStore().Delete(ctx, sbx.ID)
+				if err != nil {
+					return fmt.Errorf("failed to delete sandbox from store: %w", err)
```

Comment:
```suggestion
					return fmt.Errorf("failed to delete sandbox in response to missing metadata for sandbox %q: %w", sbx.ID, err)
```

%q should be used for the sandbox id, not %w, and also this error format is consistent with other errors below.